src/resources/views/crud/columns/upload.blade.php

```diff
@@ -0,0 +1,23 @@
+{{-- regular object attribute --}}
+@php
+    $column['value'] = $column['value'] ?? data_get($entry, $column['name']);
+    $column['text'] = $column['default'] ?? '-';
+
+    if($column['value'] instanceof \Closure) {
+        $column['value'] = $column['value']($entry);
+    }
+
+    if(is_array($column['value'])) {
+        $column['value'] = json_encode($column['value']);
+    }
+
+    if(!empty($column['value'])) {
+        $column['text'] = $column['value'];
+    }
+@endphp
+
+<span>
+    @includeWhen(!empty($column['wrapper']), 'crud::columns.inc.wrapper_start')
+        {!! $column['text'] !!}
```

Comment:
This column should probably load the text column. Before it does, it would change some column attributes:
- wrapper element becomes "a"
- wrapper href becomes the link
- we add "download" attribute to wrapper ( https://www.w3schools.com/tags/att_a_download.asp )

Reply:
Worked as requested but need to find solution for 'disk' option as that's NOT mandatory in some cases.

Reply:
Applied changes & reviewed on local. Kindly check